### DISCUSSION TITLE
[Suggestions] All game scrollbars, change the operation so that when the user clicks at a location above or below the scroll bar cursor position, the list is changed only according to the number of scenarios (or others) displayed instead of positioning the list according to the length of the list

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#2689] [Ideas]
In fheroes2, it would be nice to change the operation so that when the user clicks at a location above or below the scroll bar cursor position, the list is changed only according to the number of scenarios (or others) displayed instead of positioning the list according to the length of the list.

So, this:

![Click on scroll bar fh2 01](https://user-images.githubusercontent.com/43583994/106066652-7a503000-60fd-11eb-83fc-fe9127264005.png)

![Click on scroll bar fh2 02](https://user-images.githubusercontent.com/43583994/106066658-7d4b2080-60fd-11eb-99c0-16bf88c84e5c.png)

Instead of this:

![Click on scroll bar fh2 01](https://user-images.githubusercontent.com/43583994/106066652-7a503000-60fd-11eb-83fc-fe9127264005.png)

![Click on scroll bar fh2 03](https://user-images.githubusercontent.com/43583994/106066676-863bf200-60fd-11eb-8b89-9d9c3bce298a.png)

Basically, reproduce the Windows window operation mode.

**Comment by Branikolog** [comment]
I agree, that making scrollbars work as it was implemented in Win is the best solution.
